src/App.js

```diff
@@ -1,24 +1,196 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import logo from "./logo.svg";
+import { Menu, Grid, Image } from "semantic-ui-react";
+import "./App.css";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+    <div>
+      <header>
+        <Menu stackable>
+          <Menu.Item style={{ flexGrow: "1" }}>
+            <img
+              src="https://react.semantic-ui.com/logo.png"
+              alt="placeholder"
+            />
+          </Menu.Item>
+          <Menu.Item
+            name="home"
+            // active={activeItem === "home"}
+            // onClick={this.handleItemClick}
+            // link
+            // as={Link}
+            // to="/"
```

Comment:
Why is this commented?

Reply:
Added and commented so it is easier to implement the behavior for the menu items later as it needs routing to be installed first and a function to display the active item correctly when the view changes.

Reply:
Try not to push commented code to the repo, unless absolutely necessary, to keep the code cleaner.

Reply:
Fixed